LICENSE

```diff
@@ -178,15 +178,15 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
+      boilerplate notice, with the fields enclosed by brackets "{}"
```

Comment:
I took the LICENSE file from our example-opensource-repo: https://github.com/giantswarm/example-opensource-repo/blob/master/LICENSE

But I can't tell where this change come from.

Reply:
@giantswarm/sig-ux can you tell how it should be?

Reply:
The license is originally form this template: https://github.com/licenses/license-templates/blob/master/templates/apache.txt

The squared bracket mentioned in that template does not match the actually used curcly brackets. However, in our case there are no placeholders, so this isn't relevant at all.

Actually, the whole appendix is only meant to be applied to source code files, to every single one to be precise (which is a dubious recommendation in my eyes). I'd vote for removing the entire appendix. This would also help us get rid of the entire copyright notice and year adaption problem.

Reply:
Having said that, this is more a legal problem, not a UX problem, and I am not an expert in that.

Reply:
AND this is all completelly off the topic of this PR

Reply:
We do not have a legal department and you where the one being rather big on the topic. You initiated the whole thing so I thought it is good to get your input here. That is why I pinged SIG UX.

Reply:
When in need, we have Thea as a person who can dive into legal topics, plus we can use our external lawyer. Depending on how much confidence we need on this.

Reply:
I think checking with Thea is a good idea. I also know some people that are into the software licensing topic.

When we discussed this in @giantswarm/sig-releng the consensus was if we care about licenses we should build the validation into `architect`. So we automate this and there is less to think about as devs.

It looks like we're not clear what is needed. I can disable the extra validation I added as its not helping at the moment.

Reply:
Let's talk in the releng sync and quickly tackle the problem then.

Reply:
Both in https://github.com/licenses/license-templates/blob/master/templates/apache.txt and http://www.apache.org/licenses/LICENSE-2.0.txt the bracket part is as follow 
`boilerplate notice, with the fields enclosed by brackets "[]"`

I don't know how this change was introduced but it is wrong, so let's get rid of it.

Reply:
Fixed the bracket, discussion about LICENSE in general is out of scope here and moved to product thx to sig-releng.